src/components/DataTable/DataTable.svelte

```diff
@@ -0,0 +1,35 @@
+<script lang="ts">
+	export let tableData = [{}];
```

Comment:
I think this prop should not be optional by specifying a default value. Because this component doesn't have any purpose if the `tableData` prop is not given.

And we should add a type to this prop which should be like.

```ts
export let tableData: [key: string]: string }[];
```

Now TypeScript will warn if the prop is not given, or it is not the type that you want.

Reply:
When I add lang="postcss", but it has error.

")" is expected

If you expect this syntax to work, here are some suggestions: 
If you use less/SCSS with `svelte-preprocess`, did you add `lang="scss"`/`lang="less"` to your `style` tag? If you use SCSS, it may be necessary to add the path to your NODE runtime to the setting `svelte.language-server.runtime`, or use `sass` instead of `node-sass`. 
Did you setup a `svelte.config.js`? 
See https://github.com/sveltejs/language-tools/tree/master/docs#using-with-preprocessors for more info.svelte(css-syntax-error)